configs/peer/genesis.json

```diff
@@ -102,347 +102,6 @@
           }
         }
       },
-      {
-        "Register": {
```

Comment:
Interesting, but I don't think it makes sence.

1. It's harder to implement
2. It's harder to maintain. Users have to remember to update both `json` and `rs` files
3. Another abstraction which makes the whole system even more complicated
4. No room for runtime migrations (which I plan to implement in future). For example if new `Validator` changes the name of some token it may want to reassing it to the users who owned it's previous version.
5. Tokens and Validator are very tied, I can't see a reason to decouple them